FakeServer/GraphQL/GraphQLMiddleware.cs

```diff
@@ -44,25 +46,39 @@ public GraphQLMiddleware(RequestDelegate next, IDataStore datastore, IMessageBus
                 return;
             }
 
-            if (context.Request.Method != "POST" || !context.Request.ContentType.Contains("application/graphql"))
+            if (context.Request.Method != "POST" ||
+                !context.Request.ContentType.ContainsAny("application/graphql", "application/json"))
```

Comment:
Using this kind of helper methods is extremely good solution in bigger projects, but in a smaller project like this where people just browse through the code and it is unlikely that it will be used in many places, it is better to use methods provided by the framework. This way there is less overhead for developers to understand the code.

So could be:
```cs
private readonly string[] _allowedTypes = new[] { "application/graphql", "application/json" };

...

if (context.Request.Method != "POST" || !_allowedTypes.Any(context.Request.ContentType.Contains))
```